tests/test_wsgi.py

```diff
@@ -93,3 +93,21 @@ def test_wsgi_exc():
     client = httpx.Client(app=raise_exc)
     with pytest.raises(ValueError):
         client.get("http://www.example.org/")
+
+
+def test_wsgi_generator():
+    output = [b"", b"", b"Some content", b" and more content"]
+    client = httpx.Client(app=application_factory(output))
+    response = client.get("http://www.example.org/")
+    assert response.status_code == 200
+    assert response.text == "Some content and more content"
+    assert response.content == b"Some content and more content"
```

Comment:
Do we actually *need* to assert both the `text` and `content`? We're not testing body decoding, so I feel like asserting the `text` (like in other tests here) should be enough, WDYT?

```suggestion
```